### DISCUSSION TITLE
Latest docker-compose not working

### DISCUSSION THREAD

**Discussion by ryzam** [Budibase/budibase#4574] [General]
Hi guys I'm updating my docker-compose with latest file from budibase git as per document, but unfortunately after run the updated docker-compose file I cannot access to the builder page anymore.

 
![image](https://user-images.githubusercontent.com/597759/154794947-7b41562e-a873-470a-b574-f5f7d9d2eba1.png)

**Comment by mjashanks** [comment]
Hey @ryzam... 
1. Has your `docker-error.log` been modified recently - if so, what's inside it?
2. Are all the services in your `docker-compose.yaml` running ok?
3. Might be worth trying to clear your cookies and refreshing the page.

**Reply by ryzam** [reply]
Hi, @mjashanks 

Tq, clear the cookies solve the issue.
